NetCoreRestApi/ServiceLayer/DataTransferObjects/ProductDTO.cs

```diff
@@ -0,0 +1,12 @@
+﻿
+namespace ServiceLayer.DataTransferObjects
+{
+    public class ProductDTO
+    {
+        public string Name { get; set; }
+        public string Description { get; set; }
+        public string CategoryList { get; set; }
```

Comment:
It is not list. it is string. Rename to Categories

Reply:
Done